packages/types/src/observation.ts

```diff
@@ -0,0 +1,25 @@
+import { Expand, PickRequired } from './common'
+import { Profile } from './profile'
+import { StrapiBase } from './strapi'
+
+export type ObservationBase = {
+  content: string
+  createdDate: string
+}
+
+type ObservationRelation = {
+  profile?: Profile | null
+  creator?: Profile | null
```

Comment:
These fields should probably be required, but the API may still not return relation fields if they are not populated in the url. So you can use question marks but without null.

```suggestion
  profile?: Profile
  creator?: Profile
```